config/locales/de.yml

```diff
@@ -47,56 +47,56 @@ de:
     participate:
-      students_title: "Students"
-      students_explanation_p1: "Our students come from a variety of backgrounds. Some want to become full-time developers, whereas some would like to learn the basics of coding in a supportive environment."
-      students_explanation_p2: "Sign up to join our vibrant community and attend our workshops."
-      students_button: "Sign up as a student"
+      students_title: "Studenten"
+      students_explanation_p1: "Unsere Studenten kommen aus den unterschiedlichsten Bereichen. Einige wollen Vollzeitentwickler werden, während andere die Grundlagen des Programmierens in einem unterstützenden Umfeld erlernen möchten."
+      students_explanation_p2: "Melden Sie sich an, um unserer lebendigen Gemeinschaft beizutreten und an unseren Workshops teilzunehmen."
+      students_button: "Als Student anmelden"
       coaches_title: "Coaches"
-      coaches_explanation_p1: "We encourage new students to work on our HTML/CSS, JavaScript, Ruby, Python or Git <a href='http://tutorials.codebar.io'>tutorials</a>. We also help them understand programming concepts."
-      coaches_explanation_p2: "Therefore we are always on the lookout for more developers to join our community and help coach at our workshops."
-      coaches_button: "Sign up as a coach"
+      coaches_explanation_p1: "Wir ermutigen neue Studenten, mit unseren HTML/CSS, JavaScript, Ruby, Python oder Git <a href='http://tutorials.codebar.io'>Tutorials</a> zu arbeiten. Wir helfen ihnen auch, Programmierkonzepte zu verstehen."
+      coaches_explanation_p2: "Deshalb sind wir immer auf der Suche nach weiteren Entwicklern, die sich unserer Gemeinschaft anschließen und uns bei unseren Workshops unterstützen."
+      coaches_button: "Als Coach anmelden"
       workshop_title: "Workshops"
-      workshop_explanation_p1: "We are hosted by some amazing companies who support our cause. Our normal format is 30 minutes of socialising with food and drink, followed by a lightning talk then coding."
-      workshop_explanation_p2: Help our expansion and enable us to provide help and support to our chapters by hosting and sponsoring our workshops.
-      workshop_button: "Host a workshop"
-      workshop_email_subject: "Regarding hosting a workshop"
+      workshop_explanation_p1: "Wir werden von einigen großartigen Unternehmen beherbergt, die unsere Sache unterstützen. Unser normales Format ist ein 30-minütiges geselliges Beisammensein mit Essen und Trinken, gefolgt von einem Blitzvortrag und anschließendem Coding."
+      workshop_explanation_p2: "Helfen Sie unserer Expansion und ermöglichen Sie uns, unseren Ortsverbänden Hilfe und Unterstützung zu bieten, indem Sie unsere Workshops ausrichten und sponsern."
```

Comment:
`Ortsverband` sounds way too official. Maybe it's ok to use `Chapter` or maybe it's ok to use `Gruppe`?

Also `Expansion` does not feel right, you're not a mega corp. Maybe `Helfen Sie uns zu wachsen`?

Reply:
Fully agree to use `wachsen` instead of `Expansion`

`Helfen Sie uns zu wachsen und ermöglichen Sie uns, unseren lokalen Gruppen Hilfe und Unterstützung zu bieten, indem Sie unsere Workshops ausrichten und sponsern.`